### ISSUE TITLE
How to move the stream title, etc., to the bottom?

### ISSUE THREAD

**Issue by premiumsmart** [FrankerFaceZ/FrankerFaceZ#1596]
How can I make it appear below the player instead of above it? It happened by itself, and I don't know why. I can't find anything related to this in FFZ, but after restoring the backup from FFZ, it went back to the top

![image](https://github.com/user-attachments/assets/1f03b16e-b48c-40e5-b3ac-6a325c8c7c65)

**Comment by TsukiZero** [comment]
if it can't be fixed, at least add latency info to the bottom, near the follow button

**Comment by DarkAlice92** [comment]
Yes, I got this exact issue randomly in the past hour. Opening channels in incognito mode has the layout as normal. And clearing cache didn't fix it for normal mode.

**Comment by Berwve** [comment]
It's a new twitch update that's being pushed in waves to some users only, I'd also like a FFZ option to bring it to how it was, as currently the top one is missing the stream latency info and it's just plain ugly imo

**Comment by Primeey** [comment]
Go to Experiments and set "web_channel_metadata_layout" to control then refresh. That seems to bring it back to the default layout for now.

**Comment by jandraelune** [comment]
Not only is the title and data moved to the top, but the links for videos and clips are just gone when the streamer is live.

**Comment by premiumsmart** [comment]
> Go to Experiments and set "web_channel_metadata_layout" to control then refresh. That seems to bring it back to the default layout for now. I haven't tested this so it might break things because it's in "Experiments" after all. ![image](https://private-user-images.githubusercontent.com/8511642/394876182-c11dee5d-73f1-46bd-b18a-9d30d2dd369e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5NzQ1ODgsIm5iZiI6MTczMzk3NDI4OCwicGF0aCI6Ii84NTExNjQyLzM5NDg3NjE4Mi1jMTFkZWU1ZC03M2YxLTQ2YmQtYjE4YS05ZDMwZDJkZDM2OWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMDMzMTI4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Njg1NmQ3NmU4ZDBlZGY5ODNlZGEwYTAzN2U1OGJmYzAwMGNjMDI1ZGQ1Zjg4MzUyYjMxMDRmMDk1YzBmNGE0YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.nmZSym9zMi77ttBe5Awdtt3P3y5fJLe0MIYUnZxKI8A)

It helped, thank you!

**Comment by TsukiZero** [comment]
> Go to Experiments and set "web_channel_metadata_layout" to control then refresh. That seems to bring it back to the default layout for now. I haven't tested this so it might break things because it's in "Experiments" after all. ![image](https://private-user-images.githubusercontent.com/8511642/394876182-c11dee5d-73f1-46bd-b18a-9d30d2dd369e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjU4OTcsIm5iZiI6MTczNDAyNTU5NywicGF0aCI6Ii84NTExNjQyLzM5NDg3NjE4Mi1jMTFkZWU1ZC03M2YxLTQ2YmQtYjE4YS05ZDMwZDJkZDM2OWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTc0NjM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGQ3MTY0ODRhODI3MTNiNDczZTY4ZDQwOWI1YTg4YTBiNGIxYjI2NTFmOTNjNWQ2ZWY2YzQ4ZjcwOTc5NDU5ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.oRYcyxLqLqC1eA165J31R_aVZbo-all_xAq1_EzKe6Q)

This does, in fact, work as a good solution until it is promoted from experiment.

**Comment by TsinJu** [comment]
> Go to Experiments and set "web_channel_metadata_layout" to control then refresh. That seems to bring it back to the default layout for now. I haven't tested this so it might break things because it's in "Experiments" after all. 

Unfortunately it doesn't work anymore

**Comment by Primeey** [comment]
> > Go to Experiments and set "web_channel_metadata_layout" to control then refresh. That seems to bring it back to the default layout for now. I haven't tested this so it might break things because it's in "Experiments" after all.
> 
> Unfortunately it doesn't work anymore

It most definitely does.

**Comment by TsinJu** [comment]
> It most definitely does.

Well, had it working too, but from starting Twitch today it does not, at least for me, no clue why

**Comment by Primeey** [comment]
> > It most definitely does.
> 
> Well, had it working too, but from starting Twitch today it does not, at least for me, no clue why

I just tried it 5 minutes ago. Change the setting back to default, refresh then change it to control again and remember to refresh after.

**Comment by premiumsmart** [comment]
I confirm it works
